src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -364,6 +364,23 @@ private bool CanConnect(object parameter)
             return true;
         }
 
+        // Handler to invoke node Auto Complete
+        private void AutoComplete(object parameter)
+        {
+            DynamoViewModel dynamoViewModel = this._node.DynamoViewModel;
+            // If the feature is enabled from Dynamo experiment setting and if user interaction is on input port.
+            if (dynamoViewModel.EnableNodeAutoComplete && this.PortType == PortType.Input)
+            {
+                (dynamoViewModel.CurrentSpaceViewModel.NodePortAutoCompleteSearchViewModel as NodeAutoCompleteSearchViewModel).targetPortViewModel = parameter as PortViewModel;
+                dynamoViewModel.CurrentSpaceViewModel.ShowNodeAutoCompleteSearchCommand.Execute(ShowHideFlags.Show);
+            }
+        }
+
+        private bool CanAutoComplete(object parameter)
+        {
+            return true;
```

Comment:
I think the check for `dynamoViewModel.EnableNodeAutoComplete` should be here as that condition dictates if `CanAutoComplete` is true or false.

Reply:
But we dont have access to DynamoViewModel here?